pkg/synopsysctl/cmd_update.go

```diff
@@ -581,6 +581,12 @@ func setBlackDuckFileOwnershipJob(namespace string, name string, pvcName string,
 		},
 		Spec: batchv1.JobSpec{
 			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						"app":       "blackduck",
+						"component": "job-pod",
```

Comment:
can we have the component name as "ownership" or "change-ownership" or something meaningful?

Reply:
Actually the labels are not needed (I forgot to remove them). I just removed the labels

Reply:
:-)